ui-backend/catalog-ui-search/pom.xml

```diff
@@ -492,7 +497,8 @@
                             ows-v_1_1_0-schema,
                             jaxb2-basics-runtime,
                             catalog-rest-service,
-                            owasp-java-html-sanitizer
+                            owasp-java-html-sanitizer,
+                            catalog-rest-endpoint
```

Comment:
Usually it is bad practice to embed bundles.  I would expect that the rest endpoint classes you are trying to access would be available within the OSGI service registry and so embedding this jar shouldn't be necessary

Reply:
Agreed, we should not be embedding this bundle here as it will likely cause unintended consequences.  Also see my other comments on its use later as this shouldn't be needed.